### ISSUE TITLE
Is This Project Still Alive ?

### ISSUE THREAD

**Issue by caturandi-labs** [moebiusphp/loop#1]
Hello author, I really appreciated with this package and really like the idea. This is very easy to use. But Is this project still alive ?
when I use `Loop::await(promise);`

I got this

```
   DEPRECATED  Creation of dynamic property Moebius\Loop\Drivers\ReactDriver::$exceptionHandler is deprecated in vendor/moebius/loop/src/Drivers/ReactDriver.php on line 30.
```

Any idea ?

**Comment by frodeborli** [comment]
Hi, I was not aware of this bug. I didn't think anybody really cared about this library at all so I kind of gave up on it. Are you using it with react? You are the first one to ever open an issue with the project, so either there aren't any bugs or nobody is using it, it seems to me. But I will of course fix that bug and also consider a route for transitioning from moebius to phasync.

**Comment by caturandi-labs** [comment]
Yeah I just use this library in laravel 11 project. Very easy to use and I really impressed with your concept. I never deal using another php async library like react or amp. I use mobieus only both for coroutine and loop. Please don't give up .. 😃

**Comment by caturandi-labs** [comment]
Phasync? Woow sounds interesting...

**Comment by frodeborli** [comment]
Phasync is very similar to Moebius, but more focused and a total rewrite. I published an update for Moebius today, to get rid of that warning. I suspect you have react installed as a dependency maybe, since you are getting this error.

Sendt fra Outlook for iOS<https://aka.ms/o0ukef>
________________________________
Fra: Catur Andi Pamungkas ***@***.***>
Sendt: Tuesday, May 21, 2024 12:57:59 AM
Til: moebiusphp/loop ***@***.***>
Kopi: Frode Børli ***@***.***>; Assign ***@***.***>
Emne: Re: [moebiusphp/loop] Is This Project Still Alive ? (Issue #1)


Phasync? Woow sounds interesting...

**Comment by frodeborli** [comment]
Just following up on this older issue. As Frode mentioned, [phasync](https://github.com/phasync/phasync) is the actively maintained successor to the Moebius packages with a cleaner API and full PHP 8.1+ support.

The deprecation warning was fixed at the time, but Moebius is no longer being developed. If you're still using async PHP, phasync is the way to go!

Closing this issue. Thanks for your interest in the project!
